### ISSUE TITLE
46899682: UITableView leading swipe actions not available in VoiceOver if there are no trailing swipe actions

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#20855]
#### Description

Summary:

If a UITableView's delegate provides leading swipe actions for a cell, but returns an empty array of trailing swipe actions, the leading actions are not available for VoiceOver users.

Steps to Reproduce:

- Open attached sample project
- Run on device
- Swipe to right on cell in table view to expose leading swipe actions (Works!)
- Enable VoiceOver
- Select cell in table view. No actions are available on the cell for VoiceOver users

Expected Results:

- The leading swipe actions should be available as custom actions in VoiceOver.

Actual Results:

- The leading swipe actions are NOT available in VoiceOver, unless the cell also has trailing swipe actions.

Version/Build:

Tested on iOS 11 and iOS 12.

Configuration:

Any

-
Product Version: iOS 11, 12
Created: 2018-12-21T14:52:23.249569
Originated: 2018-12-21T08:51:00
Open Radar Link: http://www.openradar.me/46899682
